staging/src/github.com/kubeedge/api/util/flag/flags.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright 2020 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flag
```

Comment:
Is the pacakge `util` required for the kubeedge/api project?

Reply:
There is indeed a dependency reference relationship. For example, api/componentconfig/edgecore/v1alpha1 needs to reference the content in `util`.

Reply:
Ok